webapp/pages/profile/_id/_slug.vue

```diff
@@ -132,7 +132,7 @@
             <ds-space v-for="follow in uniq(user.followedBy)" :key="follow.id" margin="x-small">
               <!-- TODO: find better solution for rendering errors -->
               <client-only>
-                <user-teaser :user="follow" />
+                <user-teaser :user="subscribe" />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-22T12:04:50Z" title="Saturday, February 22nd 2020, 1:04:50 pm +01:00">Feb 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -132,7 +132,7 @@
             <ds-space v-for="follow in uniq(user.followedBy)" :key="follow.id" margin="x-small">
               <!-- TODO: find better solution for rendering errors -->
               <client-only>
-                <user-teaser :user="follow" />
+                <user-teaser :user="subscribe" />
```


This will certainly break functionality. You would need to rename the property or method in the page component with the same name.

@Ruben80 if you do a refactoring and rename all words in a code base, I suggest to use `git grep <WORD>`. It's like a full text search through your indexed files.

You can also write
> fix #3085 

in your PR description. That will automatically close the issue when this PR has been merged.

Apart from that I feel slightly nervous about #3085. It's quite a lot of files to change and there is a lot which could break. Not a "good first issue" I would say. I don't know, maybe you want to focus on something else? Would you enjoy a pair-programming with sb. from the team?